Packs/FeedThreatVault/Integrations/FeedThreatVault/FeedThreatVault.py

```diff
@@ -0,0 +1,353 @@
+        tlp_color (str): The tlp color of the indicator.
+        feed_tag_name (str): the name of the feed broken down to a tag.
+
+    Returns:
+        dict[str, Any]: An indicator that can be sent to TIM.
+    """
+    # print(tags)
+    all_tags = argToList(tags)
+    all_tags.append(feed_tag_name)
+
+    fields = assign_params(
+        indicatoridentification=indicator,
+        tags=all_tags,
+        trafficlightprotocol=tlp_color
+    )
+
+    return assign_params(
+        value=indicator,
+        type=FeedIndicatorType.IP,
+        fields=fields
+    )
+
+
+"""
+COMMANDS
+"""
+
+
+def threatvault_get_indicators_command(client: Client, list_format: str, args: Dict) -> CommandResults:
+    """Threatvault get indicators query main command.
+
+    Args:
+        client (Client): client.
+        args (dict): arguments.
+
+    Returns:
+        CommandResults: Command results response.
+    """
+    name = args.get("name")
+    version = args.get("version")
+    offset = LIMIT
+    ipaddr_list = []
+
+    query = assign_params(
+        name=name,
+        version=version,
+        listformat=list_format,
+        limit=LIMIT,
+    )
+
+    try:
+        response = client.get_indicators_request(args=query)
+
+    except DemistoException as err:
+        if err.res is not None and err.res.status_code == 404:
+            response = {}
+            readable_output = "There is no information for your search."
+            CommandResults(readable_output=readable_output)
+        else:
+            raise
+
+    if response.get('success'):
+        count = response.get("count", 0)
+        content_version_number = response.get("data", {}).get("version")
+        ipaddr_list.extend(response.get("data", {}).get("ipaddr", []))
+
+        # get next page of data until there is none left
+        while count > offset:
+
+            query = assign_params(
+                name=name,
+                version=version,
+                listformat=list_format,
+                offset=offset,
+            )
+
+            response = client.get_indicators_request(args=query)
+            ipaddr_list.extend(response.get("data", {}).get("ipaddr", []))
+            offset += LIMIT
+
+        # create the table based on the response
+        table = {
+            'Name': args.get("name"),
+            'Count': int(count),
+            'Content_Version': int(content_version_number),
+        }
+
+        markdown = tableToMarkdown(f'ThreatVault EDL Results for: {name}\n', table, removeNull=True)
+
+        table_full = table
+        table_full['Addresses'] = ipaddr_list
+
+        return CommandResults(
+            outputs_prefix=f'{INTEGRATION_ENTRY_CONTEXT}.{CONTEXT_KEY}',
+            outputs_key_field='',
+            outputs=table_full,
+            readable_output=markdown,
+            raw_response=response
+        )
+
+    else:
+        raise DemistoException(f"couldn't fetch, {response.get('success')}, response.get('message')")
```

Comment:
If we got to this section in the code, than response.get('success') is None, so what value does it add to print it?
Maybe we could print the error code or error message instead?

Reply:
Yep, thats a typo, forgot the curly brackets, fixed.